coverage/src/test/java/gov/cms/ab2d/coverage/service/CoverageServiceImplTest.java

```diff
@@ -237,10 +243,10 @@ void checkCoveragePeriodStatus() {
         assertFalse(coverageService.isCoveragePeriodInProgress(period1Jan.getId()));
 
         assertThrows(
-            CoveragePeriodNotFoundException.class,
-            () -> {
-                coverageService.isCoveragePeriodInProgress(999999999);
-            }
+                CoveragePeriodNotFoundException.class,
+                () -> {
+                    coverageService.isCoveragePeriodInProgress(999999999);
+                }
         );
```

Comment:
This also appears to be extra indentation, but maybe I am wrong and the linter actually asked you to do this?

Reply:
just auto formatting. reverted